modules/upgrade/pages/server-major-upgrade-uyuni.adoc

```diff
@@ -0,0 +1,88 @@
+[server-major-upgrade]
+= Server - Major Upgrade
+
+From time to time, {productname} Server upgrades are labeled as major.
```

Comment:
I'm wondering whether it would be handy to assign some kind of such a label to the Uyuni releases. For the time being, I'll rephrase this (done). I'd like to keep the product documentation version agnostic as much as possible.  I think it's sufficient if you'll mention the version numbers in the release notes.

Reply:
I agree, no need to mention the versions in the doc. I was just giving an example to show that, in the end, we don't have such distinction between minor an major upgrades.